CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# Code Ownership for buzzmobile.
+# Order matters. The last match is used, so more specific owners should be 
+# last.
+
+# Default owners/reviewers for anything.
+* @irapha @joshuamorton
+
+# Language specific reviewers:
+*.py @iraphael @joshuamorton
+*.cpp @chsahit
+
+# Folder and package level ownership.
+ci_scripts/* @irapha @joshuamorton
+
+# Buzzmobile subpackage overrides.
+buzzmobile/simulation/* @chsahit
+buzzmobile/tests/* @joshuamorton
```

Comment:
Yes. And that was sort of my thought. As time goes on, certain test directories would have other owners.